pygpudrive/env/base_env.py

```diff
@@ -102,7 +102,8 @@ def _setup_environment_parameters(self):
         params.IgnoreNonVehicles = self.config.remove_non_vehicles
         params.maxNumControlledVehicles = self.max_cont_agents
         params.isStaticAgentControlled = False
-        params.useWayMaxModel = False
+        params.useWayMaxModel = self.config.use_bicycle_model
+        params.useDeltaModel = self.config.use_delta_model
 
```

Comment:
Looks good. After adding in the `StateDynamics` model, I will update this such that it is not possible to select > 1 dynamics model